### ISSUE TITLE
Whipsers no longer appearing in chat log

### ISSUE THREAD

**Issue by jarrettkong** [ValveSoftware/Dota2-Gameplay#3880]
### Description

Using the `/w` command in chat works properly, but it no longer appears in the chat log. 

Steps to repo:
1. Open chat in menu or game, use `/w` command and select a user. Type a message and send.
2. Notice the recent chat log does not display the message, press `Enter` to confirm in full chat log.
3. Verify the message was sent in the Steam overlay with `Shift+Tab`.
4. Alternatively, get a reply from a user and notice the notification in the top right, not rendered as a whisper.

### Example Match ID (and possibly Timestamp)

n/a

**Comment by jarrettkong** [comment]
Appears this user may have some kind of privacy setting enabled, because other users show in the chat log for me when using whispers.

However, the auto complete does not always show all users on my friend's list. I can type their name manually to send the whisper however.

**Comment by notivago** [comment]
I don’t think it is a permission problem, I started having the same issue reported yesterday mid match, a friend was writing to me as I was playing, messages were appearing as they should and then suddenly they were not.
